docs/troubleshooting/rancher_prerequisite.md

```diff
@@ -0,0 +1,50 @@
+# Prerequisites to run CStor-CSI in rancher based clusters
+
+## Intro
+CStor-CSI provides block volume support through the iSCSI protocol. Therefore, the iSCSI client(initiator) presence on all Kubernetes nodes is required.
+
+## Step to be performed on Rancher based K8s cluster
+
+- If you are using RancherOS as the operating system, you need to enable the iSCSI service and start iSCSI service on all the worker nodes.
+- If you are using Ubuntu or RHEL as the operating system, you need to
+    - Verify if iSCSI initiators are installed on all nodes.
+    - Add the extra_binds under Kubelet service in cluster YAML file to mount the iSCSI binary and configuration inside the kubelet.
```

Comment:
Does this extra bind point need to go under RancherOS thing and not under Ubuntu?

Reply:
No, it should be under Ubuntu only. I followed [this](https://docs.openebs.io/docs/next/prerequisites.html#rancher) doc to fill these steps.

Reply:
I will verify by experimenting

Reply:
But I think I did not do any extra bind for ubuntu-based OS

Reply:
@sonasingh46 It is required for users who are using both SPC and CSPC APIs to provision volumes(If user doesn't perform extra binds then he will have problem for non-CSI based volumes).

Reply:
cc @gprasath @prateekpandey14

Reply:
I got it.